src/Orc.CsvTextEditor/Services/CsvTextEditorInstance.cs

```diff
@@ -94,6 +95,14 @@ internal class CsvTextEditorInstance : ICsvTextEditorInstance
             _textEditor.TextArea.TextView.LineTransformers.Add(new FirstLineAlwaysBoldTransformer());
 
             initializer.Initialize(textEditor, this);
+
+            _refreshViewTimer = new DispatcherTimer();
+            _refreshViewTimer.Tick += (sender, e) => {
+                RefreshView();
+                Debug.WriteLine("Refresh views");
+                (sender as DispatcherTimer).Stop();
+            };
+            _refreshViewTimer.Interval = new TimeSpan(0, 0, 0, 0, 50);
```

Comment:
TimeSpan.FromMilliseconds is easier to read, then my mind doesn't have to parse all parameters ;-)

Reply:
Hi Geerrt, You want me to change that now?

Thanks for handling these pull requests, you mind if I ask some questions about Orc.FilterBuilderControl?

Reply:
No, it's mostly for future reference. You can ask my any question, best is in gitter or on the repository itself.

Reply:
Will do.